backend/main.go

```diff
@@ -244,16 +244,20 @@ func initialize(servicePort, databaseURL, redisURL, loggingURL, statsdURL string
 func main() {
 	// get our pertinent information from the environment variables or the command line
 	servicePort := flag.String("port", os.Getenv("PORT"), "port to run server on")
-	databaseURL := flag.String("database_url", os.Getenv("DATABASE_URL"), "database url")
+	postgresURL := flag.String("postgres_url", os.Getenv("POSTGRES_URL"), "postgres url")
```

Comment:
Our database, in the future, could be anything! In future PRs we will abstract away from using pgx in initializing the database and instead just require whatever database someone uses to conform an interface.

```suggestion
	databaseURL := flag.String("database_url", os.Getenv("DATABASE_URL"), "database url")
```